Testing/functional/tests/cypress/integration/e2e/user/vaccineCardQuickLink.js

```diff
@@ -0,0 +1,123 @@
+const { AuthMethod } = require("../../../support/constants");
+
+const homePath = "/home";
+
+const encounterModule = "Encounter";
```

Comment:
Lines 5 to 10 are these constants constants being used?  If not, please remove them.